articles/service-fabric/how-to-managed-cluster-modify-node-type.md

```diff
@@ -268,7 +268,6 @@ In this walkthrough, you will learn how to modify the OS image for a node type u
 4) Select the `Node type name` you want to modify
 
 5) Adjust the `OS Image` to the new value you want and select `Apply` at the bottom. 
-![Sample showing changing the OS image][change-os-image]
```

Comment:
why not fix the image link instead? the path that is being referred to here is incorrect. This looks like a type and should instead point to the path on line 439:
[change-nodetype-os-image]: ./media/how-to-managed-cluster-modify-node-type/sfmc-change-os-image.png

ie this line should be `![Sample showing changing the OS image][change-nodetype-os-image]` instead of  `![Sample showing changing the OS image][change-os-image`